src/app/files/file-explorer.js

```diff
@@ -441,35 +441,42 @@ fileExplorer.prototype.toGist = function (id) {
     }
   }
 
+  async function getOriginalFiles (id) {
```

Comment:
Maybe add a comment with jsdoc synthax so we can understand what this function is doing : 
```
/**
 * This function is awesome
 * @params id The if of....
 */
async function getOriginalFiles (id) { ... }
```